packages/react/src/oidc/vanilla/oidc.ts

```diff
@@ -628,24 +628,29 @@ Please checkout that you are using OIDC hook inside a <OidcProvider configuratio
          timer.clearTimeout(this.timeoutId);
      }
      
-    async logoutAsync(callbackPath: string | undefined = undefined, extras:StringMap=null) {
+    async logoutAsync(callbackPath: string | undefined = undefined, extras:StringMap=null, postLogoutRedirectUri: string | undefined = undefined) {
```

Comment:
I would prefer something like that.

`
 async logoutAsync(callbackPathOrUrl: string | undefined = undefined, extras:StringMap=null) {
        const configuration = this.configuration;
        const oidcServerConfiguration = await this.initAsync(configuration.authority, configuration.authority_configuration);
        if(callbackPath && (typeof callbackPath !== 'string'))
        {
            callbackPath = undefined;
            console.warn('callbackPath path is not a string');
        }
        const path = (callbackPath === null || callbackPath === undefined) ? location.pathname + (location.search || '') + (location.hash || '') : callbackPath;
		const isUri = callbackPathOrUrl.includes("https://") || callbackPathOrUrl.includes("http://");
		let url = isUri ? callbackPathOrUrl : window.location.origin + path;
        // @ts-ignore
        const idToken = this.tokens ? this.tokens.idToken : "";
        await this.destroyAsync();  
        if(oidcServerConfiguration.endSessionEndpoint) {
            let extraQueryString = "";
            if(extras){
                for (let [key, value] of Object.entries(extras)) {
                    extraQueryString +=`&${key}=${encodeURIComponent(value)}`;
                }
            }
            window.location.href = `${oidcServerConfiguration.endSessionEndpoint}?post_logout_redirect_uri=${encodeURIComponent(url)}&id_token_hint=${idToken}${extraQueryString}`;
}
`

Reply:
I prefer that solution. I found i cleaner and simpler to use

Reply:
I like this solution.

Reply:
Thank you very much @LucassvTI. 
Awesome!